Validation/ceDigi.fcl

```diff
@@ -1,4 +1,5 @@
 #include "Production/JobConfig/digitize/OnSpill.fcl"
+physics.producers.MakeSS : { module_type : NullProducer }
```

Comment:
I'm sure this is necessary, but out of curiosity why is this needed here but not for standard CE digi-making in JobConfig?

Reply:
It's because SurfaceSteps appeared late in MDC2020 so some samples don't have them. makeSS is a placeholder module for creating them if needed. There's no one right answer, this PR switches the default from creating them to not creating them. for MDC2025 we won't need makeSS outside of primary generation.